src/components/TezosDataView.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { DataView } from '../types';
+import { TezosBlockView } from './TezosBlockView';
+import { TezosAccountView } from './TezosAccountView';
+import { TezosOperationView } from './TezosOperationView';
+
+interface TezosDataProps {
+    dataView: DataView;
+    network: string;
+    id: string;
+}
+
+interface TezosDataState {
+
+}
+
+export class TezosDataView extends React.Component<TezosDataProps, TezosDataState> {
+
+    constructor(props: TezosDataProps) {
+        super(props);
+        this.state = {};
+    }
+
+    public render() {
+        if (this.props.dataView === DataView.Blocks) {
```

Comment:
Personal preference but I think switch statements are more readable and less writing for things like this:
```
switch (this.props.dataView) {
  case (DataView.Blocks):
     return ...
```

Reply:
Makes sense. Changed!